src/homepageExperience/components/steps/Overview.tsx

```diff
@@ -13,7 +14,24 @@ export class Overview extends PureComponent {
             and write and execute some basic queries.
           </p>
 
-          <p>Without further ado, let’s get started.</p>
+          <iframe
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/KZwr1xBDbBQ"
+            title="YouTube video player"
```

Comment:
I think we should change this to something like `InfluxData - What is Time Series`

Reply:
Oh yeah good call!